introduction.md

```diff
@@ -1 +1,4 @@
 Hi I'm westi, my favourite code school path is Electives ;)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?